src/seedsigner/views/view.py

```diff
@@ -291,7 +291,7 @@ def run(self):
             title=_("Work In Progress"),
             status_headline=_("Not Yet Implemented"),
             text=self.text,
-            button_data=[ButtonOption("Back to Main Menu")],
+            button_data=[ButtonOption("Back to main menu")],
```

Comment:
Since this is referring to the name of a screen, perhaps preserve the screen's title case?

But that being said, the screen is actually called "Home".

@easyuxd: Do you think we should change this to "Home" and any thoughts on capitalization when referring to a screen by title?